tests/core/channels/test_slack.py

```diff
@@ -143,19 +145,21 @@ def test_slack_form_metadata():
     assert metadata["out_channel"] == channel
     assert metadata["users"][0] == authed_user
     assert metadata["thread_id"] == ts
+    assert metadata["team_id"] is None
```

Comment:
This appears to be one of the failing cases, line 431 in slack.py needs to be updated to return the "team_id" parameter

Reply:
Right, I'm not clear though on why this is the case. Can you check my comment above? I wonder if there is an issue deeper in the code base that doesn't allow None values to be added to the metadata. If that's the intended behavior, I can just remove that line from the test case. @b-quachtran

Reply:
The assert statement is checking that the "team_id" field exists in the metadata and that the value for that field is `None`. It's failing because this response does not include "team_id" in the JSON response as a field: 

```
        if content_type == "application/x-www-form-urlencoded":
            # if URL-encoded message is received
            output = request.form
            payload = json.loads(output["payload"][0])
            message = payload.get("message", {})
            thread_id = message.get("thread_ts", message.get("ts"))

            users = []
            if payload.get("user", {}).get("id"):
                users.append(payload.get("user", {}).get("id"))

            return {
                "out_channel": payload.get("channel", {}).get("id"),
                "thread_id": thread_id,
                "users": users,
            }
```

You'll need to update it to:
```
            return {
                "out_channel": payload.get("channel", {}).get("id"),
                "thread_id": thread_id,
                "users": users,
                "team_id": message.get("team_id"),
            }
```

Reply:
Ah, thank you. Looks like I overlooked the extraction in the case it has the "application/x-www-form-urlencoded" header. I checked the Slack API reference documentation here: https://api.slack.com/reference/interaction-payloads/block-actions and updated the form object accordingly to include this piece of metadata. It's slightly different than the above.